R/plot_model_output.R

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
Style - can you remove the leading whitespace and the Author line (eventually all files will have a great many contributors :stuck_out_tongue_closed_eyes:) and replace that with a line in `DESCRIPTION`'s `authors@R` (put yourself as `aut`)